API/testrunner/devices/rpi2.py

```diff
@@ -127,7 +127,16 @@ def execute(self, testset, test):
         # Create the command that the device will execute.
         command = template % (self.workdir, testdir, apps[self.app], testfile)
 
+        if self.env['info']['coverage'] and self.app == 'iotjs':
+            command += ' --coverage-port %s' %  utils.read_port_from_url(self.env['info']['coverage'])
+
+            # Start the client script on a different thread for coverage.
+            client_thread = Thread(target = utils.run_coverage_script,  kwargs={'env' :self.env})
```

Comment:
Please unify the style of the variable assignments (put/remove spaces around `=`) and remove the extra space before `kwargs`.